docs/admin_api/user_admin_api.rst

```diff
@@ -95,6 +95,9 @@ Body parameters:
 
```

Comment:
github won't let me comment on it, but on the line above:

> - ``deactivated``, optional, defaults to ``false``.

should be:

> - ``deactivated``, optional. If unspecified, deactivation state will be left unchanged on existing accounts and set to ``false`` for new accounts.

?

Reply:
Makes sense, yeah. 👍